tests/test_http_client.py

```diff
@@ -103,3 +118,33 @@ def test_tls_enabled(
         monkeypatch.setenv("REQUESTS_CA_BUNDLE", ca_cert_file)
 
         assert http_client.generate_text(model_name, "dummy text")
+
+
+@pytest.mark.parametrize("connection_type", [ConnectionType.MTLS], indirect=True)
+def test_client_instantiation(
+    ca_cert_file,
+    client_key_file,
+    client_cert_file,
+    connection_type,
+):
+    # MTLS tests
+    with pytest.raises(
+        ValueError,
+        match="Must provide both ca_cert_path, client_cert_path, \
+            client_key_path for mTLS",
```

Comment:
```suggestion
        match="Must provide both ca_cert_path, client_cert_path, \
            client_key_path for mTLS",
```

Should be changed to match the above exception